.github/workflows/ci.yml

```diff
@@ -253,6 +261,63 @@ jobs:
           fi
           cmake --install ./build --prefix "$llvm_project_root"/install
 
+      - name: LibC++ Binaries
+        id: libcxx-cache
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.llvm-parameters.outputs.libcxx-root }}
+          key: libcxx-${{ runner.os }}-${{ matrix.compiler }}-${{ matrix.version }}-${{ steps.llvm-parameters.outputs.llvm-hash }}
+
+      - name: Install LibC++
+        id: libcxx-install
+        if: ${{ steps.libcxx-cache.outputs.cache-hit != 'true'}}
+        shell: bash
+        run: |
+          set -x
+
+          cd ..
+          mkdir -p third-party/llvm-project
+          cd third-party/llvm-project
+          llvm_project_root=$(pwd)
+          git config --global init.defaultBranch master
+          git config --global advice.detachedHead false
+          git init
+          git remote add origin https://github.com/llvm/llvm-project.git
+          git fetch --depth 1 origin ${{ steps.llvm-parameters.outputs.llvm-hash }}
+          git checkout FETCH_HEAD
+
+          export CXX="$llvm_project_root/install/bin/clang++"
+          export CC="$llvm_project_root/install/bin/clang"
+
+          cd $llvm_project_root
+
+          if [ ${{ runner.os }} != 'Windows' ]; then
+            cmake -G Ninja  \
+                  -S runtimes  \
+                  -B build-libcxx  \
+                  -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi;libunwind"  \
+                  -D CMAKE_INSTALL_PREFIX="$llvm_project_root/install-libcxx"
+
+            ninja -C build-libcxx cxx cxxabi unwind
+            ninja -C build-libcxx install-cxx install-cxxabi install-unwind
+          else
+            cmake -G Ninja \
+                  -S runtimes \
+                  -B build-libcxx \
+                  -DLLVM_ENABLE_RUNTIMES="libcxx;libcxxabi" \
+                  -DCMAKE_INSTALL_PREFIX="$llvm_project_root/install-libcxx" \
```

Comment:
I’ll use `$llvm_project_root/install/libcxx` for installing libc++, to avoid conflicts with existing LLVM source directories

Reply:
The question is: do any conflicts exist?

Reply:
I do not know... I am doing some tests right now to confirm it.
I  also moved LLVM installation to `$llvm_project_root/install/llvm`

Reply:
You'll need to update CI and the documentation then. CMake LLVM_ROOT is set to llvm_root, and llvm_root is set to ../third-party/llvm-project/install. 

Regarding the other problem: 

> The question is: do any conflicts exist?
> I do not know...

What you are doing is creating two Filesystem Hierarchy Standard (FHS) trees: one for LLVM and one for libcxx. 
But when you install LLVM and libcxx in your machine, you're installing it in a single FHS tree: /usr/local or /usr.

So you do know. There are no conflicts. There can't be any conflicts or otherwise it wouldn't be possible to install them in a single tree.

In principle, there's no problem having separate trees. In practice, however, that means your CMake script now has too look for two things: LLVM and libcxx, because they're not in the same tree. If they are in the same tree, you can find libcxx relative to the LLVM include directory. If they're not in the same tree, you need to give CMake this tree: some kind of LIBCXX_ROOT and find libcxx from there. And there's no built-in package to do that.

Reply:
Done